client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -63,6 +65,23 @@ function ReaderSubscriptionListItem( {
 		return <ReaderSubscriptionListItemPlaceholder />;
 	}
 
+	function recordEvent( name ) {
+		const props = {
+			blog_ID: siteId,
+			feed_ID: feedId,
+		};
+		if ( railcar ) {
+			recordTrackWithRailcar( name, railcar, props );
+		} else {
+			recordTrack( name, props );
+		}
+	}
+
+	const recordTitleClick = () => recordEvent( 'calypso_reader_feed_link_clicked' );
+	const recordAuthorClick = () => recordEvent( 'calypso_reader_author_link_clicked' );
+	const recordSiteUrlClick = () => recordEvent( 'calypso_reader_site_url_clicked' );
+	const recordAvatarClick = () => recordEvent( 'calypso_reader_avatar_clicked' );
```

Comment:
while testing this, noticed that we _don't_ report avatar clicks elsewhere in calypso? odd.